src/gfx.cpp

```diff
@@ -681,9 +681,7 @@ int DrawString(int left, int right, int top, std::string_view str, TextColour co
  */
 int DrawString(int left, int right, int top, StringID str, TextColour colour, StringAlignment align, bool underline, FontSize fontsize)
 {
-	char buffer[DRAW_STRING_BUFFER];
-	GetString(buffer, str, lastof(buffer));
-	return DrawString(left, right, top, buffer, colour, align, underline, fontsize);
+	return DrawString(left, right, top, GetString(str), colour, align, underline, fontsize);
```

Comment:
worth marking these functions as inline? I imagine the compiler probably is already, but...

Reply:
Don't you have to move the functions to the header to make them inline? Then we'd be adding a dependency on `strings_func.h` to `gfx_func.h`.
I'm not sure whether it's worth doing that. I'm personally in favour of letting the compiler decide (during LTO) whether to inline it, instead of the whole spiel with adding hints that it might/will ignore and increasing compilation time due to the dependency on `strings_fun.h`.